user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -12,7 +12,7 @@ Supported versions:
 - OpenSearch 2.X
 
 > [!NOTE]
-> OpenSearch is only supported on Linux.
+> OpenSearch is supported on Windows from OpenSearch 2.4 onwards.
```

Comment:
Hi @RobbeDGH, OK to merge it like this?

Reply:
i rather just remove it as I dont see a reason why people would still download 2.3 or lower

Reply:
(context: we are already at 2.9)

Reply:
If OpenSearch can only be used on Windows from a certain version onwards, then I agree with Edith that we should mention that "as from" version.

Reply:
I agree that it is unlikely for people to download an OpenSearch version prior to 2.9 today. However, it is possible that some users might still be using an older version they had previously installed. Since our documentation explicitly states that we offer support for these older versions, it would be more thorough and user-friendly to retain the note for their benefit.

Reply:
Good for me, feel free to merge